randovania/games/prime1/logic_database/Tallon Overworld.txt

```diff
@@ -616,13 +616,28 @@ Extra - aabb: [-576.05975, -221.71631, -188.35965, -170.50539, 225.37242, 45.510
       Trivial
   > Event - Meta Ridley
       All of the following:
-          Artifact of Chozo and Artifact of Elder and Artifact of Lifegiver and Artifact of Nature and Artifact of Newborn and Artifact of Spirit and Artifact of Strength and Artifact of Sun and Artifact of Truth and Artifact of Warrior and Artifact of Wild and Artifact of World and Shoot Any Beam
-          Charge Beam or Combat (Intermediate)
+          Have all Artifacts
```

Comment:
You are missing a change in the header file to define this new template. I imagine you lost the change somewhere when recreating the branch.